From fcb164dca80b69a81d8f8ab5c2999df9ecb40f1d Mon Sep 17 00:00:00 2001
From: "sp.mridu" <sp.mridu@enflame-tech.com>
Date: Tue, 3 Sep 2024 06:10:27 +0000
Subject: [PATCH] This is a Commit to test for generating a ptach

---
 setup.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/setup.py b/setup.py
index eeccfa7..ca9ce5b 100755
--- a/setup.py
+++ b/setup.py
@@ -2,6 +2,7 @@
 
 #Newline added
 # Newline added from the browser
+# A Line for Patch
 
 from setuptools import setup, find_packages
 import platform
@@ -30,6 +31,7 @@ elif platform.system() != 'Linux':
         'Please contact the author if you know any virtual joystick driver for your system.')
 
 # https://stackoverflow.com/questions/20288711/
+# A Line for Patch
 from setuptools.command.install import install
 class PostInstallCommand(install):
     # vJoy_url cannot be cached from the previous conditionals.
@@ -44,6 +46,7 @@ class PostInstallCommand(install):
                 'If you need to install this driver, you can do so anytime by visiting ' + vJoy_url)
             answer = ctypes.windll.user32.MessageBoxW(0,
                 'Yoke was installed succesfully, but can only work if the correct vJoy driver is installed. '
+ 
                 'The driver for Windows ' + platform.release() + ', version ' + platform.version() + ' can be found at:\n\n' + vJoy_url + '\n\n'
                 'Click OK if you want to download the installer now, or Cancel otherwise.',
                 'vJoy driver required', 33) # question prompt (33), OK (answer == 1) and Cancel (answer == 2) buttons
@@ -51,6 +54,7 @@ class PostInstallCommand(install):
                 import webbrowser
                 webbrowser.open_new(vJoy_url)
         install.run(self)
+# A Line for Patch
 
 setup(
     name='yoke',
-- 
2.37.3

